### PR TITLE
Update NameAddrHeader.js

### PR DESCRIPTION
**Pull request by shevtsov-max**
https://github.com/versatica/JsSIP/issues/782

### CODE REVIEW

File: lib/NameAddrHeader.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = class NameAddrHeader
   constructor(uri, display_name, parameters)
   {
     // Checks.
-    if (!uri || !(uri instanceof URI))
+    if (!uri || (URI && !(uri instanceof URI)))
```
Comment (ibc) [inline lib/NameAddrHeader.js:27]: This is not gonna fix anything. If `URI` doesn't exist in window/global then `if (URI)` will throw anyway. 

This is not a valid fix. 

The problem is that `URI.js` loads `Utils.js` and `Utils.js` also loads `URI.js` so at the time `URI.js` is loaded it's empty. This is a problem we had in the past and requires a mayor refactor. This PR is not fixing it at all so I'm closing it.

Comment (shevtsov-max) [reply lib/NameAddrHeader.js:27]: In order to use the library, I need this fix. I did it through my profile if you can't do this check. I'm not the only one with the problem, and it's been a long time, judging by the open issue.

Comment (ibc) [reply lib/NameAddrHeader.js:27]: I've been very specific in my comment above and you are not saying anything about it.

Comment (shevtsov-max) [reply lib/NameAddrHeader.js:27]: You've given a good answer, but it doesn't solve the problem right now. You wrote "This is a problem we had in the past and requires a mayor refactor", but when will it be available? I'm offering a temporary solution that will help now, and at least many people will be able to run the library. And then you can solve the problem as you described.

Comment (ibc) [reply lib/NameAddrHeader.js:27]: What I meant is that your workaround cannot properly work despite it seems that it works. The problem is that, using Vue require() implementation there is a deadlock situation and one of the calls to require("./URI.js") is literally importing nothing and that's why you get undefined URI.

**Pull request closed without merging**